pandas/core/internals/blocks.py

```diff
@@ -231,11 +231,12 @@ def get_values(self, dtype=None):
             return self.values.astype(object)
         return self.values
 
-    def get_block_values(self, dtype=None):
+    def get_block_values_for_json(self) -> np.ndarray:
         """
-        This is used in the JSON C code
+        This is used in the JSON C code.
         """
-        return self.get_values(dtype=dtype)
+        # TODO(2DEA): reshape will be unnecessary with 2D EAs
```

Comment:
Putting such comments in several places won't make this happen ... (if you want this, you will need to lead a discussion about it)

Reply:
experience shows that discussion turns on amounts of complexity; marking these will allow us to pseudo-quantify that

Reply:
agree with both of you! these do have good placeholders, but am skeptical of 2D EA.